client/apollo/react/src/Form/Radio/RadioCommon.tsx

```diff
@@ -0,0 +1,50 @@
+import React, {
+  ReactNode,
+  forwardRef,
+  useId,
+  type ComponentType,
+  type ComponentProps,
+} from "react";
+import { ItemMessage } from "../ItemMessage/ItemMessageCommon";
+
+type RadioProps = {
+  label: string | ReactNode;
```

Comment:
pas besoin de mettre string

```suggestion
  label: ReactNode;
```